### DISCUSSION TITLE
RemoteLight server on Raspberry Pi 4

### DISCUSSION THREAD

**Discussion by ry-ol** [Drumber/RemoteLight#39] [Q&A] [answered]
Hi, I was wondering how the performance was on a Raspberry Pi 4. I currently have a Raspberry Pi Zero W that I set up with 300 LEDs and it is unusably slow. I know the Pi 4 has a quad core, just wondering if that makes a significant difference. I plan to add more lights to the same device in the future (upwards of 900).

**Comment by Drumber** [comment, marked as answer]
Hi, if you are using RemoteLightWeb there should definitely be a big performance gain with the Pi 4 as it also has more RAM. However, it shouldn't make much of a difference if you only use RemoteLightServer with the desktop app.
